kateryna/databases.py

```diff
@@ -0,0 +1,113 @@
+class PostgreSQLConnectionRefusedError(Exception):
+    def __init__(self, message):
+        self.message = message
+
+
+class Connection(object):
+    def __init__(self, db_name):
+        self.closed = False
+        self.db_name = db_name
+
+    def __str__(self):
+        return "Connection to {}".format(self.db_name)
+
+
+class BaseFramework(object):
```

Comment:
It would be also great to have `__enter__` and `__exit__` methods to make it possible to use our database connector to use as a context manager.